src/routers/incompleteApplication.js

```diff
@@ -0,0 +1,45 @@
+const express = require('express')
+const IncompleteApplication = require('../models/IncompleteApplicationModel')
+const router = new express.Router()
+const authMiddleware = require('../middleware/authMiddleware')
+
+router.put('/editing', authMiddleware, async (req, res) => {
+  console.log(req.body)
+  try {
+    const existingIncompleteApp = await IncompleteApplication.findOne({ owner: req.user._id })
+    if (existingIncompleteApp) {
+      const updates = Object.keys(req.body)
+      const allowedUpdates = ['name', 'email', 'phoneNumber', 'university', 'major', 'classYear', 'shirtSize', 'ethnicity', 'gender', 'pronouns', 'github', 'linkedin', 'personalSite', 'otherSite', 'originCity', 'campusParking', 'travelSponsorship', 'validAge', 'dietaryRestrictions', 'allergies', 'referral', 'personalProject', 'hackathonGoal', 'mlhSignature']
+      const isValidOperation = updates.every((update) => allowedUpdates.includes(update))
+      if (!isValidOperation) {
+        return res.status(400).send({ error: 'Invalid updates' })
+      }
+      for (var update of updates) {
```

Comment:
Don't use var, maybe consider let inside if you're using this method to loop. Look into the .forEach loop for clear syntax